src/domain/occurrences/fields/OccurrenceAttendedField.tsx

```diff
@@ -54,8 +54,4 @@ const OccurrenceAttendedField = () => {
   );
 };
 
-OccurrenceAttendedField.defaultProps = {
-  label: 'enrolments.fields.attended.label',
-};
-
```

Comment:
Same as in https://github.com/City-of-Helsinki/kukkuu-admin/pull/293/commits/eef44481cb03cd8c3ddf1a091f52c1a19bb0a4b7#r1890156393